developer/src/kmcmplib/src/CompMsg.cpp

```diff
@@ -0,0 +1,129 @@
+    { CERR_StringInVirtualKeySection  , "exted string illegal in virtual key section"},
+    { CERR_AnyInVirtualKeySection  , "'any' command is illegal in virtual key section"},
+    { CERR_InvalidAny         , "Invalid 'any' command"},
+    { CERR_StoreDoesNotExist  , "Store referenced does not exist"},
+    { CERR_BeepInVirtualKeySection  , "'beep' command is illegal in virtual key section"},
+    { CERR_IndexInVirtualKeySection  ,  "'index' command is illegal in virtual key section"},
+    { CERR_BadCallParams      , "CompileKeyboardFile was called with bad parameters"},
+    { CERR_InfileNotExist     , "Cannot find the input file"},
+    { CERR_CannotCreateOutfile, "Cannot open output file for writing"},
+    { CERR_UnableToWriteFully , "Unable to write the file completely"},
+    { CERR_CannotReadInfile   , "Cannot read the input file"},
+    { CERR_SomewhereIGotItWrong , "Internal error: contact Keyman"},
+    { CERR_BufferOverflow     , "The compiler memory buffer overflowed"},
+    { CERR_Break              , "Compiler interrupted by user"},
+    { CERR_CannotAllocateMemory , "Out of memory"},
+    { CERR_InvalidBitmapLine  , "Invalid 'bitmaps' command"},
+    { CERR_CannotReadBitmapFile , "Cannot open the bitmap or icon file for reading"},
+    { CERR_IndexDoesNotPointToAny  , "An index() in the output does not have a corresponding any() statement"},
+    { CERR_ReservedCharacter  , "A reserved character was found"},
+    { CERR_InvalidCharacter   , "A character was found that is outside the valid Unicode range (U+0000 - U+10FFFF)"},
+    { CERR_InvalidCall        , "The 'call' command is invalid"},
+    { CERR_CallInVirtualKeySection  , "'call' command is illegal in virtual key section"},
+    { CERR_CodeInvalidInKeyStore  , "The command is invalid inside a store that is used in a key part of the rule"},
+    { CERR_CannotLoadIncludeFile  , "Cannot load the included file: it is either invalid or does not exist"},
+    { CERR_60FeatureOnly_EthnologueCode  , "EthnologueCode system store requires VERSION 6.0"},
+    { CERR_60FeatureOnly_MnemonicLayout  , "MnemonicLayout functionality requires VERSION 6.0"},
+    { CERR_60FeatureOnly_OldCharPosMatching  , "OldCharPosMatching system store requires VERSION 6.0"},
+    { CERR_60FeatureOnly_NamedCodes  , "Named character constants requires VERSION 6.0"},
+    { CERR_60FeatureOnly_Contextn , "Context(n) requires VERSION 6.0"},
+    { CERR_501FeatureOnly_Call , "Call() requires VERSION 5.01"},
+    { CERR_InvalidNamedCode   , "Invalid named code constant"},
+    { CERR_InvalidSystemStore , "Invalid system store name found"},
+    { CERR_CallIsProfessionalFeature  , "Call() is only available in Keyman Developer Professional"},
+    { CERR_IncludeCodesIsProfessionalFeature  , "IncludeCodes is only available in Keyman Developer Professional"},
+    { CERR_MnemonicLayoutIsProfessionalFeature  , "MnemonicLayout is only available in Keyman Developer Professional"},
```

Comment:
Are these still used or could they be removed?

Reply:
OK